hw/ip/rv_core_ibex/doc/_index.md

```diff
@@ -87,17 +88,31 @@ The Ibex crash dump state includes the following information:
 * The next PC
 * The last data access address
 * The last exception address
+* The last exception PC
```

Comment:
Could be worth clarifying these are `mtval` and `mepc` to avoid any confusion.

Reply:
o yeah good idea.